### DISCUSSION TITLE
How to use QField without QGIS

### DISCUSSION THREAD

**Discussion by janbolspfizer** [opengisch/QField#5529] [Q&A]
I'm looking into ways to setup a QField project using a number of layers without generating the project inside QGIS. I want to avoid having to use QGIS to do a field assessment using QField. 

Instead I would like to use some custom developed solution to do that. I was thinking about using QFieldCould's API to generate the project and add the necessary geopackages as files. 

Is such a thing doable do you think? Are there other people doing similar things?

**Comment by m-kuhn** [comment]
You should be able to use a [standalone python application](https://docs.qgis.org/3.34/en/docs/pyqgis_developer_cookbook/intro.html#python-applications) to create the .qgs project and then use the [qfieldcloud sdk](https://docs.qfield.org/reference/qfieldcloud/sdk/) to create / sync a cloud project.
